lib/ansible/plugins/connection/winrm.py

```diff
@@ -249,8 +249,8 @@ def _reset(self):  # used by win_reboot (and any other action that might need to
 
     def exec_command(self, cmd, in_data=None, sudoable=True):
         super(Connection, self).exec_command(cmd, in_data=in_data, sudoable=sudoable)
-        cmd_parts = shlex.split(to_bytes(cmd), posix=False)
-        cmd_parts = map(to_text, cmd_parts)
+        cmd_parts = shlex.split(to_text(cmd), posix=False)
+        cmd_parts = list(map(to_text, cmd_parts))
```

Comment:
shlex.split needs to use bytes on python2 (more specifically, on python-2.6.  It's a bug that wasn't fixed until python-2.7).  So you can use a conditional on six.PY3 to change this.

In this (and indeed, in many cases), map is better done as a list comprehension.  So:

``` python
cmd_parts = [to_text(p) for p in cmd_parts]
```